examples/train.py

```diff
@@ -126,6 +126,9 @@ def run_model_stablebaseline(flow_params,
     stable_baselines.*
         the trained model
     """
+    from stable_baselines.common.vec_env import DummyVecEnv, SubprocVecEnv
+    from stable_baselines import PPO2
```

Comment:
@eugenevinitsky can these imports (and the ones below) be removed? It seems they aren't called, and that's causing the flake8 tests to fail.